core/src/main/java/org/polypheny/db/catalog/entity/logical/LogicalNamespace.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2019-2023 The Polypheny Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.polypheny.db.catalog.entity.logical;
+
+
+import io.activej.serializer.annotations.Deserialize;
+import io.activej.serializer.annotations.Serialize;
+import java.io.Serial;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.Value;
+import lombok.With;
+import lombok.experimental.NonFinal;
+import org.polypheny.db.catalog.Catalog;
+import org.polypheny.db.catalog.entity.PolyObject;
+import org.polypheny.db.catalog.logistic.DataModel;
+import org.polypheny.db.type.entity.PolyString;
+import org.polypheny.db.type.entity.PolyValue;
+
+
+@EqualsAndHashCode(callSuper = false)
+@With
+@Value
+@NonFinal // for testing
+public class LogicalNamespace implements PolyObject, Comparable<LogicalNamespace> {
+
+    @Serial
+    private static final long serialVersionUID = 3090632164988970558L;
+
+    @Serialize
+    public long id;
+    @Serialize
+    @Getter
+    public String name;
+    @Serialize
+    @Getter
+    @EqualsAndHashCode.Exclude
+    public DataModel dataModel;
+    @Serialize
+    public boolean caseSensitive;
```

Comment:
Can you please expose this to the UI and the DDLs.

Reply:
Can you elaborate what you mean by this?

Reply:
There is not yet an option in the UI to set and adjust the case sensitivity of a namespace. We should add this. Further, we should also add it to the ddls for creating namespaces in SQL, MQL and cypher and add ddls to modify case sensitivity of existing namespaces.